Sources/Testing/Issues/Issue.swift

```diff
@@ -405,3 +405,54 @@ extension Issue.Kind {
     }
   }
 }
+
+// MARK: - Snapshot CustomStringConvertible, CustomDebugStringConvertible
+
+extension Issue.Snapshot: CustomStringConvertible, CustomDebugStringConvertible {
+  public var description: String {
```

Comment:
It's unfortunate we have so much duplicated code. Any ideas about how to reduce it would be helpful.